challenge-1/exercise-3/exercise-3-follow-the-mode.js

```diff
@@ -2,9 +2,36 @@
 const array1 = [1, 1, 2, 3, 4, 5, 5, 6, 7, 8, 8, 8, 9];
 
 // Second scenario
-const array2 = [0, 0, 5, 1, 6, 7, 6, 6, 7, 2, 0, 8, 8];
+const array2 = [0, 0, 5, 1, 6, 7, 6, 6, 7,7,7, 2, 0, 8, 8];
 
 // Third scenario
 const array3 = [0, 1, 2, 3, 4, 5, 6, 7, 8];
 
+// Fourth scenario
+const array4 = [0,0,0,0,1,1,1,1,1,1,1,2,3];
+
 // code here...
```

Comment:
const array = [1,1,2,3];

const dict = {
 1: 2,
 2: 1,
 3: 1,
}

for (let i = 1; i < array.length; i++) {
  const currentNumber = array[i]

  if (dict[currentNumber]) {
     dict[currentNumber]++
     return;
  }

  dict[currentNumber] = 1
}

// array